client/src/storage/index.js

```diff
@@ -1,6 +1,5 @@
-'use strict';
-
-var eventsData = require("json!./events.json");
+const Backbone = require('backbone');
+const eventsData = require('./events.json');
```

Comment:
This import is now broken. It needs to load with the json loader, or the webpack config needs to be setup to handle json files with the json loader.